Library/Homebrew/cmd/audit.rb

```diff
@@ -512,9 +512,9 @@ def audit_github_repository
     end
 
     problem "GitHub fork (not canonical repository)" if metadata["fork"]
-    if (metadata["forks_count"] < 10) && (metadata["subscribers_count"] < 10) &&
-       (metadata["stargazers_count"] < 20)
-      problem "GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)"
+    if (metadata["forks_count"] < 50) && (metadata["subscribers_count"] < 50) &&
+       (metadata["stargazers_count"] < 100)
```

Comment:
I'm not sure if we want to be so harsh on *all* other projects just because someone tries to game our system every once in a while (very rarely, I guess). This seems like too much collateral damage to me. I'm leaning more towards blacklisting the few individual projects/users that we notice instead.

The commit message claims to **double** the numbers, but the factor is actually **five**. :wink: